v2/internal/frontend/desktop/windows/frontend.go

```diff
@@ -689,6 +696,73 @@ func (f *Frontend) processMessage(message string) {
 	}()
 }
 
+func (f *Frontend) processMessageWithAdditionalObjects(message string, sender *edge.ICoreWebView2, args *edge.ICoreWebView2WebMessageReceivedEventArgs) {
+	if strings.HasPrefix(message, "file:") {
+		callbackID := message[5:]
+
+		objs, err := args.GetAdditionalObjects()
+		defer objs.Release()
```

Comment:
This should also be moved after the `if err != nil` block, so we don't panic if `GetAdditionalObjects` has returned an error.